src/omotes_simulator_core/entities/assets/pipe.py

```diff
@@ -47,23 +47,42 @@ class Pipe(AssetAbstract):
     output: List[Dict[str, float]]
     """The output list of the pipe with a dictionaries for each timestep."""
 
-    def __init__(self, asset_name: str, asset_id: str, port_ids: list[str]):
+    def __init__(
+        self,
+        asset_name: str,
+        asset_id: str,
+        port_ids: list[str],
+        length: float,
+        inner_diameter: float,
+        roughness: float,
+        alpha_value: float,
+        minor_loss_coefficient: float,
+        external_temperature: float,
+        qheat_external: float,
+    ):
         """Initialize a Pipe object.
 
         :param str asset_name: The name of the asset.
         :param str asset_id: The unique identifier of the asset.
         :param List[str] port_ids: List of ids of the connected ports.
+        :param float length: The length of the pipe.
```

Comment:
For reference add the SI-units in the docstring and remove the docstring from above to below. These variables have changed from non user-defined variables to user-defined variables.

Reply:
Completed.